internal/config/main.go

```diff
@@ -73,6 +79,63 @@ func (wiz configWizard) deserializeConfigYaml(
 	return &config, nil
 }
 
+func (wiz configWizard) readSecretFromFile(filePath string) (string, error) {
+	secretBytes, err := os.ReadFile(filePath)
+	if err != nil {
+		return "", err
```

Comment:
I'd prefer some context for this error, to help debugging:
```suggestion
		return "", fmt.Errorf("unable to read secret from %s: %w", filePath, err)
```

Reply:
I added the message